src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+
+table.tHead.addEventListener('click', (ev) => {
+  const item = ev.target;
+  const cell = item.cellIndex;
+  const row = [];
+
+  for (let i = 1; i < table.rows.length - 1; i++) {
+    const values = [];
+
+    for (let j = 0; j < table.rows[0].cells.length; j++) {
+      const value = table.rows[i].cells[j].innerText;
+
+      values.push(value);
```

Comment:
`Values` and `value` can be easily confused. Also, they don't give information about what exactly value they consist.